src/Magnum/Platform/EmscriptenApplication.h

```diff
@@ -0,0 +1,1509 @@
+             *
+             * If set, the WebGL power preference will be set to reduce power
+             * consumption.
+             */
+            PreferLowPowerToHighPerformance,
+
+            /**
+             * Fail if major performance caveat
+             *
+             * If set, requests context creation to abort if the browser is
+             * only able to create a context that does not give good hardware-
+             * accelerated performance.
+             */
+            FailIfMajorPerformanceCaveat,
+
+            /**
+             * Explicit swap control
+             *
+             * For more details, see the
+             * [Emscripten API reference](https://emscripten.org/docs/api_reference/html5.h.html#c.EmscriptenWebGLContextAttributes.explicitSwapControl)
+             * for more details.
+             */
+            ExplicitSwapControl,
+
+            /**
+             * Enable WebGL extensions by default
+             *
+             * For more details, see the
+             * [Emscripten API reference](https://emscripten.org/docs/api_reference/html5.h.html#c.EmscriptenWebGLContextAttributes.enableExtensionsByDefault)
+             * for more details.
+             */
+            EnableExtensionsByDefault,
+
+            /**
+             * Render via offscreen back buffer
+             *
+             * For more details, see the
+             * [Emscripten API reference](https://emscripten.org/docs/api_reference/html5.h.html#c.EmscriptenWebGLContextAttributes.renderViaOffscreenBackBuffer)
+             * for more details.
+             */
+            RenderViaOffscreenBackBuffer,
+
+            /**
+             * Proxy content to main thread
+             *
+             * For more details, see the
+             * [Emscripten API reference](https://emscripten.org/docs/api_reference/html5.h.html#c.EmscriptenWebGLContextAttributes.proxyContextToMainThread)
+             * for more details.
+             */
+            ProxyContextToMainThread,
+        };
+
+        /**
+         * @brief Context flags
+         *
+         * @see @ref setFlags(), @ref Context::Flags
+         * @requires_gles Context flags are not available in WebGL.
+         */
+        //typedef Containers::EnumSet<Flag, Flag::PremultipliedAlpha> Flags;
+        typedef Containers::EnumSet<Flag> Flags;
+
+        /*implicit*/ GLConfiguration();
+
+        /**
+         * @brief Context flags
+         *
+         * @requires_gles Context flags are not available in WebGL.
+         */
+        Flags flags() const { return _flags; }
+
+        /**
+         * @brief Set context flags
+         * @return Reference to self (for method chaining)
+         *
+         * Default is @ref Flag::ForwardCompatible on desktop GL and no flags
+         * on OpenGL ES.
+         * @see @ref addFlags(), @ref clearFlags(), @ref GL::Context::flags()
+         * @requires_gles Context flags are not available in WebGL.
+         */
```

Comment:
This doc block is ... not making any sense ;) Similarly the `@requires_gles` part in all related functions below.